cid/helpers/quicksight/__init__.py

```diff
@@ -764,12 +849,37 @@ def describe_user(self, username: str) -> dict:
             logger.info(f'QuickSight user {username} not found.')
             return None
 
+    def describe_group(self, groupname: str) -> dict:
+        """ Describes an AWS QuickSight Group """
+        try:
+            result = self.identityClient.describe_group(**{
+                'AwsAccountId': self.account_id,
+                'GroupName': groupname,
+                'Namespace': 'default'
+            })
+            logger.debug('group = ',json.dumps(result))
+            return result.get('Group')
+        except self.client.exceptions.ResourceNotFoundException:
+            logger.info(f'QuickSight group {groupname} not found.')
+            return None
+        except self.client.exceptions.AccessDeniedException: # Try to overcome AccessDeniedException
+            #FIXME: paginator is not available for list_groups
+            groups = self.identityClient.list_groups(AwsAccountId=self.account_id, Namespace='default').get('GroupList')
```

Comment:
This can throw an exception too

Reply:
i guess we do not need this workaround here. simplified.